xla/service/gpu/autotuning/conv_algorithm_picker_test.cc

```diff
@@ -36,13 +37,27 @@ limitations under the License.
 #include "tsl/platform/statusor.h"
 #include "tsl/platform/test.h"
 
+#if GOOGLE_CUDA
+#include "third_party/gpus/cuda/include/cuda.h"
```

Comment:
This requires a corresponding dependency in the BUILD file guarded with if_cuda_is_configured